lib/functional-constraints/searchOrCreateKeys.js

```diff
@@ -11,14 +11,15 @@ const validateSearchOrCreateKeys = (definition) => {
   const errors = [];
   const searchKeys = _.keys(definition.searches);
   _.each(definition.searchOrCreates, (searchOrCreateDef, key) => {
```

Comment:
It is below in the error message, to reference which `searchOrCreate` is this for.